internal/pkg/service/buffer/worker/service/cleanup.go

```diff
@@ -0,0 +1,90 @@
+package service
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/distribution"
+)
+
+type cleanup struct {
+	*Service
+	logger   log.Logger
+	assigner *distribution.Assigner
```

Comment:
Este pozeram, ze `assigner` tu mas ulozeny ale nikde sa nepouziva.

Myslim, ze by si ho mal pouzit v tom `for`, kde iterujes `receivers`, a tak zistit, ci dany receiver ma riesit tato node, alebo je to uloha inej node.